src/corelibs/U2Core/src/gobjects/MultipleChromatogramAlignmentObject.h

```diff
@@ -70,10 +71,21 @@ class U2CORE_EXPORT MultipleChromatogramAlignmentObject : public MultipleAlignme
     void deleteColumnsWithGaps(U2OpStatus &os);
 
     void trimRow(const int rowIndex, int currentPos, U2OpStatus &os, TrimEdge edge);
+
+    /**
+      * Updates the corresponding alternative mutations.
+      * Shows the second strongest character, if it's peak height persantage is more then @settings.threshold.
```

Comment:
Data object should not mess with visual mode here: 'shows' is related  to the view state, not to the object state.

Reply:
Ok